lib/gssapi/spnego/init_sec_context.c

```diff
@@ -655,7 +677,7 @@ spnego_reply(OM_uint32 * minor_status,
 	}
     }
 
-    if (*resp.u.negTokenResp.negState != accept_completed ||
+    if (negState != accept_completed ||
```

Comment:
See above comment.  Just because `negState == incomplete` doesn't mean we need to send a reply.  If we set it to `incomplete` by inference, and we don't have a token to send back, and we're locally-complete, then we should not send back a reply!

Reply:
We could probably use a test.  In `tests/gss/check-context.in` we could try to accept a token for a name for which we don't have a keytab entry.

Reply:
OK, let me think about this some more :)

Reply:
See above, you're right :)

Reply:
See above comments – original patch was correct.